types/ws/index.d.ts

```diff
@@ -70,101 +70,106 @@ declare class WebSocket extends EventEmitter {
         options?: WebSocket.ClientOptions | ClientRequestArgs,
     );
 
-    close(code?: number, data?: string | Buffer): void;
-    ping(data?: any, mask?: boolean, cb?: (err: Error) => void): void;
-    pong(data?: any, mask?: boolean, cb?: (err: Error) => void): void;
-    send(data: any, cb?: (err?: Error) => void): void;
+    close(code?: number, data?: WebSocket.Data): void;
+    ping(data?: WebSocket.Data | number, mask?: boolean, cb?: (err: Error) => void): void;
+    pong(data?: WebSocket.Data | number, mask?: boolean, cb?: (err: Error) => void): void;
+    send(data: WebSocket.Data | number | ArrayLike<any>, cb?: (err?: Error) => void): void;
```

Comment:
[The docs](https://github.com/websockets/ws/blob/8525ec95729c63a67b36e67a4f9b9d12af9abdd1/doc/ws.md#websocketsenddata-options-callback) say that `DataView` and `Object` are also valid options for `ws.send`, `ws.ping`, and `ws.pong`